main.py

```diff
@@ -2,6 +2,50 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    # Create
+    Genre.objects.create(name="Western")
```

Comment:
I would strongly advise using loops here.

Reply:
Agree w u, thanks a lot. I forgot that we can use loops here.